contracts/kernel/KernelStorage.sol

```diff
@@ -16,4 +16,5 @@ contract KernelConstants {
 
 contract KernelStorage is KernelConstants {
     mapping (bytes32 => address) public apps;
+    bytes32 public defaultVaultId;
```

Comment:
Why do we need this extra storage?

Reply:
To recover it from here:
https://github.com/aragon/aragonOS/pull/281/files#diff-14a0803379fe8527d47915e3727fe7d0R79
and here:
https://github.com/aragon/aragonOS/pull/281/files#diff-807af7e85f274f9138c01132b86256b6R19